transform/app/tasks/utils/save_error_log.py

```diff
@@ -0,0 +1,28 @@
+import json
+import logging
+from pathlib import Path
+
+logger = logging.getLogger(__name__)
+
+
+def save_error_log_to_json(error_log: dict) -> None:
```

Comment:
Very nice idea to create such a function! I am up for it, it will be very useful. Maybe just use `LOG_FILE_NAME` cut the last `"transformer.log"` and create a separate directory name with current timestamp and put report there

Reply:
Implemented